include/zephyr/debug/mipi_stp_decoder.h

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DEBUG_MIPI_STP_DECODER_H__
+#define ZEPHYR_INCLUDE_DEBUG_MIPI_STP_DECODER_H__
+
+#include <zephyr/kernel.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup mipi_stp_decoder_apis STP Decoder API
+ * @ingroup coresight_apis
```

Comment:
@nordic-krch we don't have such a doxygen group in Zephyr upstream -- was this an oversight? Can you please fix this by either adding `mipi_stp_decoder_apis` to an existing Zephyr group, or by defining a new one?